### PR TITLE
Added Linkedin Project

### PR DESCRIPTION
**Pull request by SyedImtiyaz-1**
Issue No. #2595 

Hello Project Admin,
I have added the Linkedin Clone {Replica}

### Screenshot -
Screenshot from 2023-06-23 17-56-53

### It is ready to Merge 👍🏻

### CODE REVIEW

**Comment by SyedImtiyaz-1** [comment]
## Hello Project Admin,
Kindly give this a review .

**Comment by SyedImtiyaz-1** [comment]
Hello @TusharKesarwani ,
please review this one.

**Comment by SyedImtiyaz-1** [comment]
Hello @TusharKesarwani ,
Kindly review this 👍🏻

**Comment by SyedImtiyaz-1** [comment]
Please review this. 
@TusharKesarwani , @pranjay-poddar , @Saurabh-kumar09 , @MohitGupta121

**Comment by SyedImtiyaz-1** [comment]
Hello @MohitGupta121 ,
I have fixed the directory and added drive imgs files. 
### It is  ready to merge 👍🏻

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@SyedImtiyaz-1 PTAL

File: Js-Games/Spider_Man_Go/images/knife.png
[the remark this answers is not in the document]
Comment (SyedImtiyaz-1) [reply Js-Games/Spider_Man_Go/images/knife.png:1]: Okay Got It working on it.

**Comment by SyedImtiyaz-1** [comment]
@MohitGupta121 please review this.

**Comment by SyedImtiyaz-1** [comment]
@MohitGupta121 ,
Conflict Resolved 👍🏻

**Comment by MohitGupta121** [comment]
You said added LinkedIn clone but I saw in your PR it's Spider man game. Make new clean PR.

**Comment by MohitGupta121** [comment]
Also use proper PR template.

**Comment by SyedImtiyaz-1** [comment]
Sure

**Pull request closed without merging**